### ISSUE TITLE
Intermittent iSCSI SendTargets test failure

### ISSUE THREAD

**Issue by jimharris** [spdk/spdk#779]
I've seen this same failure multiple 
http://10.102.17.206/results/autotest-per-patch/builds/29817/archive/iscsi_autotest/build.log (on GP3)

Relevant part of the failure:
```
  22:11:23	# free_mb=762348
  22:11:23	# echo 762348
 22:11:23	# free_mb=762348
 22:11:23	# '[' 762348 -gt 2048 ']'
 22:11:23	# /var/jenkins/workspace/iSCSI_tests/iscsi_autotest/spdk/scripts/rpc.py construct_lvol_bdev -u 26172924-c015-411d-a301-e370f98ca378 lbd_0 2048
b21a78ea-443e-4c7a-a572-2c0cab2bd601
 22:11:23	# /var/jenkins/workspace/iSCSI_tests/iscsi_autotest/spdk/scripts/rpc.py construct_target_node Target1 Target1_alias lvs_0/lbd_0:0 1:2 256 -d
 22:11:23	# sleep 1
 22:11:24	# iscsiadm -m discovery -t sendtargets -p 10.0.0.1:3260
iscsiadm: Could not stat /var/lib/iscsi/nodes//,3260,-1/default to delete node: No such file or directory
iscsiadm: Could not add/update [tcp:[hw=,ip=,net_if=,iscsi_if=default] 10.0.0.1,3260,1 iqn.2016-06.io.spdk:Target1]
10.0.0.1:3260,1 iqn.2016-06.io.spdk:Target1
 22:11:24	# iscsiadm -m node --login -p 10.0.0.1:3260
iscsiadm: No records found
  22:11:24	# trap - ERR
  22:11:24	# print_backtrace
  22:11:24	# [[ ehxBE =~ e ]]
 ```

Somehow the iscsiadm database is getting corrupted.

**Comment by jimharris** [comment]
Here's another one of the failures.  Again on GP3:

http://10.102.17.206/results/autotest-per-patch/builds/29805/archive/iscsi_autotest/build.log

**Comment by jimharris** [comment]
And another one on GP3:

http://10.102.17.206/results/autotest-per-patch/builds/29799/archive/iscsi_autotest/build.log

**Comment by jimharris** [comment]
@Seth5141 and I just looked at Jenkins history on iscsi_autotest and last 6 times this test ran on GP3, it failed.  It hasn't failed on any other systems in that amount of time.

Still debugging.

**Comment by jimharris** [comment]
The iscsiadm database was corrupted on GP3.  I think I've fixed it for now.  The /var/lib/iscsi/nodes directory had a bunch of entries, including empty "default" files which seem to cause the issue shown:

https://groups.google.com/forum/#!topic/open-iscsi/bbiAW3ft7zE

I removed those empty default files for now.  Logging into GP3, that at least let me run iscsiadm -m node -o delete.

@Seth5141 remembered some commands Daniel Verkamp had put into the Chandler test pool that would clean these up on reboots.  This wasn't pulled over to the Jenkins test pool though.  Seth's adding that now.
